### PR TITLE
Update CoreCLR to 24022

### PR DESCRIPTION
**Pull request by gregg-miskelly**

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
@chuckries please review
@DustinCampbell do you need to update OmniSharp also?
@MattGertz FYI

File: src/coreclr-debug/main.ts [hunk 1]
```diff
@@ -263,28 +263,28 @@ function createProjectJson(targetRuntime: string): any
             emitEntryPoint: true
         },
         dependencies: {
-            "Microsoft.VisualStudio.clrdbg": "14.0.25214-preview-2934931",
+            "Microsoft.VisualStudio.clrdbg": "14.0.25225-preview-2934931",
```
Comment (chuckries) [inline src/coreclr-debug/main.ts]: this needs to be 14.0.25225-preview-**2954453**

**Comment by chuckries** [comment]
LGTM just fix the version of the clrdbg package.

**Comment by MattGertz** [comment]
Pre-approved.

**Pull request merged**